src/Sulu/Bundle/WebsiteBundle/Resources/views/Sitemap/sitemap.xml.twig

```diff
@@ -4,7 +4,7 @@
 {% for site in sitemap if site.locale in locales %}
 {% if site.nodeType == 1 %}
 {% set siteLocale = site.locale %}
-{% set url = sitemap_url(site.url, site.locale) %}
+{% set url = sitemap_url(site.url, site.locale, site.webspaceKey) %}
```

Comment:
it works only without webspacekey it when sitemap_url is called from Browser not CLI. 
Without the requestanalyzer is used but there is not request!